cli/inspect.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// InspectCommand is used to implement 'inspect' command.
+type InspectCommand struct {
+	baseCommand
+}
+
+// Init initializes InspectCommand command.
+func (p *InspectCommand) Init(c *Cli) {
+	p.cli = c
+	p.cmd = &cobra.Command{
+		Use:   "inspect [container]",
+		Short: "get the detailed information of container",
+		Args:  cobra.ExactArgs(1),
```

Comment:
Please add Long and Example here like create command https://github.com/alibaba/pouch/blob/master/cli/create.go#L25-L34:
```
cc.cmd = &cobra.Command{
		Use:   "create [image]",
		Short: "Create a new container with specified image",
		Long:  createDescription,
		Args:  cobra.MinimumNArgs(1),
		RunE: func(cmd *cobra.Command, args []string) error {
			return cc.runCreate(args)
		},
		Example: createExample(),
	}
```
These things are merged this morning and useful to auto generate markdown document, so you may have no idea about that now.